recipes/hlslpp/all/conanfile.py

```diff
@@ -17,6 +19,16 @@ class HlslppConan(ConanFile):
     settings = "os", "arch", "compiler", "build_type"
     no_copy_source = True
 
+    @property
+    def _min_cppstd(self):
+        return 11
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "gcc": "6",
+        }
```

Comment:
```suggestion
        return {"gcc": "6"} if Version(self.version) >= "3.3" else {}
```

Do not restrict older versions, in case they work with Conan 1.x